src/components/Calculator.js

```diff
@@ -2,46 +2,110 @@ import React from 'react';
 import Container from 'react-bootstrap/Container';
 import Row from 'react-bootstrap/Row';
 import Col from 'react-bootstrap/Col';
+import Button from 'react-bootstrap/Button';
+import calculate from '../logic/calculator';
+
 // eslint-disable-next-line react/prefer-stateless-function
 class Calculator extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      total: null,
+      next: null,
+      operation: null,
+    };
+    this.getAnswer = this.getAnswer.bind(this);
+  }
+
+  getAnswer = (e) => {
+    const newState = calculate(this.state, e.target.value);
+    this.setState(newState);
+    const showAnswer = document.querySelector('.answer');
+    if (newState.next !== null) {
+      showAnswer.innerHTML = newState.next;
+    } else {
+      showAnswer.innerHTML = newState.total;
+    }
+  }
```

Comment:
- [ ]  When an operator is clicked twice it gives an error on the browser. Kindly implement an error handling method. 

![Screenshot from 2021-08-20 03-43-32](https://user-images.githubusercontent.com/64894927/130148561-77e7cacc-6b08-4e89-bbef-e0eff291ed0a.png)

- [ ] When a number is divided by zero, it gives a browser error. Kindly implement an error handling method. 

![Screenshot from 2021-08-20 03-49-09](https://user-images.githubusercontent.com/64894927/130148955-8d6e2581-1c30-4755-9fb9-a4b28e92e735.png)

Reply:
Thanks for the review, I have resolved the errors

Reply:
This issue is well resolved! :clap:

Reply:
I have fixed it again, please test it. If you press an operator twice it displays 'undefined'. If you also press an operator before entering a number it displays 'undefined' too